src/app/shared/view-evidences/view-evidences.component.css

```
@@ -39,17 +39,17 @@
 
   ::ng-deep .mat-mdc-tab-group, 
```

Biome (2.4.7)

[error] Unexpected unknown pseudo-elements: ng-deep

(lint/correctness/noUnknownPseudoElement)